jans-cedarling/cedarling/src/init/cedar_schema.rs

```diff
@@ -8,6 +8,7 @@
 use base64::prelude::*;
 
 #[derive(Debug, thiserror::Error)]
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
What do you think about using camel case for `enums` since that's the standard practice in Rust? These are no longer constants.

Reply:
It was constants, but @duttarnab asked to do it in this way… So for me, it is actually constants…
And I prefer to move this strings to constants....

Reply:
... but OK, I will rename...

Reply:
Done